### PR TITLE
[Localization IT] Modification some code about stat changes.

### PR DESCRIPTION
**Pull request by fedesauro22**
## What are the changes?
I've changed some texts about stat changes.

## Why am I doing these changes?
I translated in Italian for a better user experience 

## What did change?
Texts about stat changes.

### CODE REVIEW

**Comment by torranx** [comment]
please include locale code in your title so its easier to distinguish

File: src/locales/it/battle.ts [hunk 1]
```diff
@@ -62,12 +62,12 @@ export const battle: SimpleTranslationEntries = {
...
-  "statRoseDrastically": "rose drastically",
-  "statWontGoAnyHigher": "won't go any higher",
-  "statFell": "fell",
-  "statHarshlyFell": "harshly fell",
-  "statSeverelyFell": "severely fell",
-  "statWontGoAnyLower": "won't go any lower",
+  "statRose": "aumenta!",
+  "statSharplyRose": "aumenta di molto!",
+  "statRoseDrastically": "aumenta moltissimo!",
+  "statWontGoAnyHigher": "non può aumentare ulteriormente",
```
Comment (CodeTappert) [inline src/locales/it/battle.ts:68]: I found this in the offical game files for this.

```
non può aumentare di più!
```

Comment (CodeTappert) [reply src/locales/it/battle.ts:68]: At https://abcboy101.github.io/poke-corpus/#query=go+any+higher&regex=false&caseInsensitive=true&common=true&script=true&collections=DiamondPearl%2CPlatinum%2CHeartGoldSoulSilver%2CBlackWhite%2CBlack2White2%2CXY%2COmegaRubyAlphaSapphire%2CSunMoon%2CUltraSunUltraMoon%2CLetsGoPikachuLetsGoEevee%2CSwordShield%2CBrilliantDiamondShiningPearl%2CLegendsArceus%2CScarletViolet%2CColosseum%2CXD%2CBattleRevolution%2CBox%2CRanch%2CBank%2CHOME&languages=de-DE%2Cen-US

File: src/locales/it/battle.ts [hunk 2]
```diff
@@ -62,12 +62,12 @@ export const battle: SimpleTranslationEntries = {
...
+  "statFell": "diminuisce!",
+  "statHarshlyFell": "diminuisce di molto!",
+  "statSeverelyFell": "diminuisce moltissimo!",
```
Comment (CodeTappert) [inline src/locales/it/battle.ts:71]: The offical game files have this as 

```
cala drasticamente
```

File: src/locales/it/battle.ts [hunk 3]
```diff
@@ -62,12 +62,12 @@ export const battle: SimpleTranslationEntries = {
...
+  "statWontGoAnyLower": "non può diminuire ulteriormente",
```
Comment (CodeTappert) [inline src/locales/it/battle.ts:72]:
```
non può diminuire di più
```

**Comment by CodeTappert** [comment]
@fedesauro22 I added some comments. BUT i am not an italian speaker. I simply went with what is in the offical game files according to https://abcboy101.github.io/poke-corpus

File: src/locales/it/battle.ts (hunk 2 above)
Comment (CodeTappert) [reply src/locales/it/battle.ts:71]: You did not do this one. Is there a reason?

Comment (fedesauro22) [reply src/locales/it/battle.ts:71]: Sorry my fault!

**Review by CodeTappert — APPROVED** [review]
All Text changed here is not the same as in the orignal games

**Comment by EnochG1** [comment]
Maybe this is translated at 
https://github.com/pagefaultgames/pokerogue/pull/2373

**Comment by Tempo-anon** [comment]
Closing for now because of merge conflicts

**Pull request closed without merging**